commons/src/main/java/org/infinispan/commons/marshall/ValueMatcherMode.java

```diff
@@ -9,6 +9,8 @@
    MATCH_EXPECTED,
    MATCH_EXPECTED_OR_NEW,
    MATCH_NON_NULL,
-   MATCH_NEVER
+   MATCH_NEVER;
+
+   public static final ValueMatcherMode[] CACHED_VALUES = values();
```

Comment:
Could we not make this public? It looks weird in the auto-completion list.  I'd rather add a public method that does the lookup, i.e. `public static ValueMatcherMode valueOf(int ordinal)`.